assignments/assignment02.rb

```diff
@@ -32,20 +31,21 @@ def to_sentence(ary)
 
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
-  sum = ary.reduce(0) {|x, acc| acc + x}
-  sum.to_f / ary.length
+  # your implementation here
+  ary.reduce(:+) / ary.length
```

Comment:
you need to convert one of the to a floating point, so that the result will be a floating point:
`ary.reduce(:+) / ary.length.to_f`